meson.build

```diff
@@ -440,6 +448,22 @@ executable('ratbagd',
 
 install_man('ratbagd/ratbagd.8')
 
+#### ratbagd_devel ####
+
+executable('ratbagd_devel',
+	   src_ratbagd,
+	   dependencies : deps_ratbagd,
+	   include_directories : include_directories('src'),
+	   install : false,
+	   c_args : '-DRATBAG_DBUS_INTF="ratbag_devel1_@0@"'.format(ratbagd_sha),
+)
+
+config_ratbagd_devel = configuration_data()
+config_ratbagd_devel.set('ratbagd_sha', ratbagd_sha)
```

Comment:
I'd say rename this to config_ratbagd_sha and then re-use it in both configure_files below